Source/Particles/PhysicalParticleContainer.H

```diff
@@ -47,6 +47,25 @@ public:
                            amrex::Real t, amrex::Real dt) override;
 #endif // WARPX_DO_ELECTROSTATIC
 
+    /**
+     * \brief Apply external E and B fields on the particles. The E and B
+     * fields could be defined as a constant or using a parser for reading
+     * in a mathematical expression. The default value for the E- and B-fields
+     * is (0.0,0.0,0.0).
+     *
+     * \param Exp-Bzp: fields on particles modified based on external E and B
+     * \param xp,yp,zp: arrays of particle positions required to compute
+     *                  mathematical expression for the external fields
+     *                  using parser.
+     */
```

Comment:
The `/** */` syntax allows Doxygen to interpret this piece of code.
However, I am not sure `\param Exp-Bzp` can be interpreted by Doxygen. Could you check by compiling the Doxygen doc? see e.g., https://warpx.readthedocs.io/en/latest/developers/documentation.html
If needed, could you fix the syntax?

Reply:
Thank you. I have now modified the param definition to \param[in,out] param_name description. Compiled and it works fine.